docs/reference/ccr/overview.asciidoc

```diff
@@ -102,9 +141,71 @@ Using these APIs in tandem enables you to adjust the read and write parameters
 on the follower shard task if your initial configuration is not suitable for
 your use case.
 
+[float]
+=== Leader index retaining operations for replication
```

Comment:
If the  "mechanics of replication" section is turned into a separate page, I think this info about how to retain operations might be a good fit there too.

Reply:
I have added a task to the meta issue #35975